docker-compose.yml

```diff
@@ -271,6 +275,42 @@ services:
     networks:
       mle_net:
 
+  # Simulator 4: Read images from Tiled using a saved URL and send via ZMQ
+  sim_local_tiled:
+    image: ghcr.io/als-computing/arroyosas:main
+    container_name: sim_local_tiled
+    command: python -m arroyosas.app.local_tiled_sim_cli
+    profiles:
+      - sim_local_tiled
+    volumes:
+      - ./data:/data:rw  # Mount directory for the URL file
+    environment:
+      - URL_FILE=/data/tiled_url/Nafion_p25_30_run_1.json
+      - TILED_API_KEY=${TILED_SINGLE_USER_API_KEY}
+      - TILED_LIVE_API_KEY=${TILED_SINGLE_USER_API_KEY}
+      - DYNACONF_TILED_POLLER__ZMQ_FRAME_PUBLISHER__ADDRESS=tcp://0.0.0.0:5000
+    networks:
+      mle_net:
+
+  # Image Ingestion Service - run this first to prepare data for Simulator 4
+  ingest_local_images:
```

Comment:
This is a note for future us: We may want to replace this with [mlex_tiled_ingest](https://github.com/mlexchange/mlex_tiled_ingest). This integration is not quite ready yet though.